src/app/core/create-store.js

```diff
@@ -0,0 +1,29 @@
+
+/**
+ * Dependencies.
+ */
+
+var createStore = require('fluxible/utils/createStore');
+var extend = require('extend');
+
+/**
+ * Generic hydrate and dehydrate methods.
+ */
+
+var genericHydration = {
+  rehydrate: function (state) {
+    this.state = state;
+  },
+  dehydrate: function () {
+    return this.state;
+  }
+};
+
+/**
+ * An extension of fluxible's createStore method.
+ */
+
+module.exports = function (storeSpec) {
+  extend(false, storeSpec, genericHydration);
```

Comment:
From the npm documentation for [extend](https://www.npmjs.com/package/extend):

> Warning: passing false as the first argument is not supported

What was this meant to do? Does that control deep copy or something?